### DISCUSSION TITLE
Suricata eve.JSON for ElasticSIEM

### DISCUSSION THREAD

**Discussion by veraxcyber** [Security-Onion-Solutions/securityonion#4088] [Unsupported Versions]
Hello all, 

How do I enable the eve.JSON file so that the data can be pushed to my elastic SIEM?

The file is not located in /var/log/suricata/eve.JSON. 

Do I need to enable it first before it will show in this directory?

Many Thanks!

**Comment by veraxcyber** [comment]
I cannot seem to find any Suricata directories but am able to see the alerts in the local host SIEM

**Reply by darth-drew** [reply]
https://github.com/Security-Onion-Solutions/securityonion/discussions/3513
https://docs.securityonion.net/en/2.3/logstash.html#forwarding-events-to-an-external-destination

**Reply by veraxcyber** [reply]
Is it possible to edit the yaml file to generate eve.json and just pull directly off the host as I am using Elastic Agents instead of needing to setup event forwarding. Thanks
